### PR TITLE
mailmap: Fan Yang information and affiliation

### PR DESCRIPTION
**Pull request by YangFanlinux**
Signed-off-by: Fan Yang <yangf@neunn.com>

### CODE REVIEW

File: .organizationmap [hunk 1]
```diff
@@ -331,6 +331,7 @@ MSys Technologies <contact@msystechnologies.com> Rajesh Nambiar <rajesh.n@msyste
 Nebula <info@nebula.com> Anton Aksola <anton.aksola@nebula.fi>
 Nebula <info@nebula.com> Chris Holcombe <chris.holcombe@nebula.com>
 Netease <contact@netease.com> Dong Wu <archer.wudong@gmail.com>
+Neunn <service@neunn.com> Fan Yang<yangf@neunn.com>
```
Comment (liewegas) [inline .organizationmap]: missing space after Yang

**Comment by joscollin** [comment]
@YangFanlinux The second commit is not signed. Please squash the commits into one too.

**Comment by YangFanlinux** [comment]
sorry for my mistake. I will create a new pr.

**Comment by joscollin** [comment]
@YangFanlinux Not necessary, you could fix it in the same PR.

**Comment by joscollin** [comment]
@YangFanlinux 
https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Pull request closed without merging**